### PR TITLE
Make port knocking whitelist gc schedule configurable

### PR DESCRIPTION
**Pull request by mixua**

### CODE REVIEW

**Comment by alexander-medved** [comment]
@max-lobur Could you please review?

File: tasks/services.yml [hunk 1]
```diff
@@ -44,7 +44,7 @@
      cron:
        name: "iptables knockd cleaner"
        state: present
-       minute: "0"
-       hour: "6"
-       weekday: "6" # Sat
-       job: 'iptables-save | grep {% for p in secure_ports %} -e "--dport {{ p[0] }} -j ACCEPT" {% endfor %} | sed -e "s/-A /-D /" | xargs -t -I {} bash -c "iptables {}"'
\ No newline at end of file
+       minute: {{port_knocking_cron.minute}}
```
Comment (max-lobur) [inline tasks/services.yml]: knockd_iptables_gc_cron

Comment (alexander-medved) [reply tasks/services.yml]: @max-lobur Changed

File: vars/main.yml [hunk 2]
```diff
@@ -1 +1,5 @@
 ---
```
Comment (max-lobur) [inline vars/main.yml]: @alexander-medved https://github.com/verygood-ops/ansible-port-knocking/blob/master/defaults/main.yml

Comment (alexander-medved) [reply vars/main.yml]: Done

Comment (alexander-medved) [reply vars/main.yml]: @max-lobur Could you please merge my PR?

**Pull request merged**